docs/graphcache/computed-queries.md

```diff
@@ -192,6 +192,30 @@ options in here `limitArgument` and `offsetArgument` these will default to `limi
 and `skip` respectively. This way you can use the keywords that you are using in
 your queries.
 
+You can also specify `mergeMode`, which defaults to `after` and can otherwise
+be set to `before`. This will handle how pages are merged when you paginate
```

Comment:
```suggestion
We may also add the `mergeMode` option, which defaults to `'after'` and can otherwise
be set to `'before'`. This will handle in which order pages are merged when paginating
```

We usually use "we" as often as possible when writing (with some exceptions). I realise the Graphcache pages may need some revamping to make sure they're as helpful as can be compared to some others, but let's take that into account here